.github/workflows/tests-aws.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests with AWS
+on:
+  push:
+    branches:
+      - master      
+    paths:
+      - '**.scala'
+      - '**.sbt'
+  workflow_dispatch:
+
+env:
+  AWS_REGION: us-east-1
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Cache Docker images
+        uses: ScribeMD/docker-cache@0.5.0
+        with:
+          key: docker-${{ runner.os }}-${{ hashFiles('docker-compose-tests.yml') }}
+      - uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
```

Comment:
I thought we compile with jdk 11

Reply:
wait, ok, it was blocked due to spark 2.x
ok, so maybe we can switch to jdk 11 with spark 3.x

Reply:
This is not really important to be support Java 8, but if we can, why not do it? Using Java 8 in the CI ensures that we at least support Java 8. On the other hand, that could also prevent us from detecting possible incompatibilities with higher versions of the JVM…
But anyway, that should be a separate issue/PR :)